.github/workflows/master.yml

```diff
@@ -0,0 +1,89 @@
+name: walrus
+on:
+  push:
+    # implicit OR for all below
+    branches: [master]
+    tags:
+      - v*.*.*
+    # If the Docker build below wasn't a "FROM scratch" container, we would
+    # want to re-build daily to ensure dependencies in the container are
+    # upgraded periodically, even if the app code does not change.
+    # ...But this is a "FROM scratch" container, so we only really have
+    # to worry about Go (the toolchain) vulnerabilities. Meh?
+    #schedule:
+    #  - cron: '0 0 * * *' 
```

Comment:
my preference would be to never ship automatically-upgraded builds

Reply:
The current implementation will build a new container with the `latest` tag for each push to master. Once I get the tagging sorted out, `latest` will point to the latest tagged release.

Reply:
I would disagree re: the value of automatic updates/rebuilds if the `Dockerfile` wasn't created the way it is, but the underlying concern (OS packages with vulns in the container) is more an ops problem than a dev problem.